nucypher/characters/unlawful.py

```diff
@@ -159,29 +159,3 @@ def circumvent_safegaurds_and_grant_without_paying(self, *args, **kwargs):
         """
         with patch("nucypher.policy.policies.Policy._publish", self.grant_without_paying):
             return self.grant_without_paying(*args, **kwargs)
-
-    def grant_while_paying_the_wrong_nodes(self,
```

Comment:
Possible to reuse this test with SubscriptionManager?...maybe not.

Reply:
SubscriptionManager facilitates payment on a network-wide basis and Alice no longer pays specific nodes.  The is another test in this module that does test for a sinpa attack against `SubscriptionManager`.

Reply:
👍